spec/functional/resource/yum_package_spec.rb

```diff
@@ -308,84 +336,84 @@ def pkg_arch
     context "with constraints" do
       it "with nothing installed, it installs the latest version" do
         flush_cache
-        yum_package.package_name("chef_rpm >= 1.2")
```

Comment:
These specs need to come back since these are valid forms of a package name for yum

Reply:
@tas50 Agreed on this we should not break which was working earlier.